src/backup/selectors.ts

```diff
@@ -3,7 +3,7 @@ import { Screens } from 'src/navigator/Screens'
 import { RootState } from 'src/redux/reducers'
 import { ONE_DAY_IN_MILLIS } from 'src/utils/time'
 
-export const shouldForceBackupSelector = (state: RootState) => {
+export const pastForcedBackupDeadlineSelector = (state: RootState) => {
```

Comment:
renamed to make it clearer that this is only one thing that should be considered to determine whether we should show the force backup screen

Reply:
(the other thing being the experiment parameter)